UKHomeOffice/sgmr-service
# Cg 121 people manifest

NOTES : 
- do `rm -rf node_modules package-lock.json` and `npm i`
- updated to govuk-frontend latest version
- there is a bug when following the 'edit voyage' path (started by clicking on a vessel name on /reports), it currently does not persist the correct voyageId. This will be fixed in a separate PR.

----

TO TEST

**I can select an existing person/people and add them to my report**
> Make sure you have created people on /people
- go to /reports
- start a new report (start now)
- fill out the report up to the /people page. You must enter a vesselname, everything else is optional
- check one or more of your people
- click Add to report
> You should see those people listed under 'People currently on board' in the bottom section of the page
- save and continue
- continue past responsible person page with save and continue (you do not need to add info)
> On the confirm page you should see your people in the Details of persons on board section

----

**I can add a new person to this report**
- go to /reports
- start a new report (start now)
- fill out the report up to the /people page. You must enter a vesselname, everything else is optional
- click on the 'Add a new person to the report' link
- fill out this form and click add to manifest
> You should see this person listed under 'People currently on board'- save and continue
- continue past responsible person page with save and continue (you do not need to add info)
> On the confirm page you should see your people in the Details of persons on board section

----
